homework/.ipynb_checkpoints/Homework1_alex_mrvaljevich-checkpoint.ipynb

```diff
@@ -0,0 +1,714 @@
+    "* Read about some [Markdown Techniques](http://daringfireball.net/projects/markdown/syntax)\n",
+    "* Write a summary of 2 chapters of [The Data Science Handbook](http://www.thedatasciencehandbook.com/) in Markdown and submit a pull request in the Lab Directory\n",
+    "\n",
+    "#### Programming\n",
+    "* Complete the lab from class and the additional Exercise below\n",
+    "\n",
+    "#### Course Project\n",
+    "* Come up with 5 different ideas for your course project. For each one list:\n",
+    "  * Overview of your idea\n",
+    "  * What data you will use\n",
+    "  * What the outcome is that you are trying to achieve\n",
+    "  * Any ideas of modelling techniques it may involve\n",
+    "\n",
+    "**Instructions: copy this file and append your name in the filename, e.g. Homework1_ian_hansel.ipynb.\n",
+    "Then commit this in your local repository, push it to your github account and create a pull request so I can see your work. Remeber if you get stuck to look at the slides going over Fork, Clone, Commit, Push and Pull request.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "# Homework 1 - Analyzing the analyzers:\n",
+    "\n",
+    "1. Analyzing the Analizers 5 key point\n",
+    "\n",
+    "    * There is a lot of confusion about what a Data Scientist actually is, and it can cause a lot of trouble for both organizations and professionals\n",
+    "    * Clustering on self-reported personality traits is an awesome approach, keen to give it a twirl!\n",
+    "    * T-shaped skills are an interesting approach, i have to work on my statistics skills\n",
+    "    * As i mainly contract, i skew personally more towards Jack-of-all-trades, it's interesting to see it's representation in the data set.\n",
+    "    * As happened with Product Management 10 years ago, Data Science is a new role and we don't yet have a [Good Data Scientist / Bad Data Scientist](http://www.khoslaventures.com/wp-content/uploads/Good_Product_Manager_Bad_Product_Manager_KV.pdf) framework.  \n",
+    "    \n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "# Homework 2 - 5 Project ideas:\n",
+    "\n",
+    "## Finding ideal thresholds for quantitative \"natural language\" metrics:\n",
+    "\n",
+    "> There is a wealth of quantitative metrics for web analytics (Bounce Rate, Time on Site, Pageviews) but none for the content portrayed on those sites. The objective of this project is to find natural language metrics that matter to outcomes (Likelihood To Read, Time on Site, Conversions) and then build a model to define ideal thresholds for a web property.\n",
+    "\n",
```

Comment:
This an interesting idea. It seems quite exploratory and open ended, which would make it a more challenging project. Also the current lesson plan is to cover NLP later in the course, so you should start playing with NLTK now. Maybe restrict this down to something like building a predictive model from  sentiment analysis, and topic clustering.